topics/admin/tutorials/ansible/tutorial.md

```diff
@@ -147,9 +149,9 @@ A `tasks/main.yml` file calls multiple Ansible modules to accomplish its goal. A
 
 Here we have two tasks. Each has a `name` that will be shown to the person running the playbook.
 
-The first invokes the `yum` module with the arguments `name: cvmfs, state: ...`. This will use yum to install the package named `cvmfs`. The state parameter uses a [Jinja2](http://jinja.pocoo.org/) template to evaluate the value of the variable `cvmfs_upgrade_client`. We can [grep through](https://github.com/galaxyproject/ansible-cvmfs/search?q=cvmfs_upgrade_client&unscoped_q=cvmfs_upgrade_client) the repository and see that `defaults/main.yml` sets that to `false` by default. We can override this if we need, we'll come back to that later. The first task also has a `when` condition to ensure it only runs on RHEL family machines, so RedHat or CentOS. It is better to use the OS-generic [`package`](http://docs.ansible.com/ansible/2.9/package_module.html) module, if possible.
+The first invokes the [`yum`](https://docs.ansible.com/ansible/2.9/modules/yum_module.html) module with the arguments `name: cvmfs, state: ...`. This will use yum to install the package named `cvmfs`. The state parameter uses a [Jinja2](http://jinja.pocoo.org/) template to evaluate the value of the variable `cvmfs_upgrade_client`. We can [grep through](https://github.com/galaxyproject/ansible-cvmfs/search?q=cvmfs_upgrade_client&unscoped_q=cvmfs_upgrade_client) the repository and see that `defaults/main.yml` sets that to `false` by default. We can override this if we need, we'll come back to that later. The first task also has a `when` condition to ensure it only runs on RHEL family machines, so RedHat or CentOS. It is better to use the OS-generic [`package`](https://docs.ansible.com/ansible/2.9/modules/package_module.html) module, if possible.
```

Comment:
The "Install CernVM-FS client (yum)" task above has been substantially changed in the `ansible-cvmfs` role and doesn't use `cvmfs_upgrade_client` directly any more. I'd suggest to replace that part of the example with:

```yaml
- name: Download cvmfs_preload utility when desired
  get_url:
    url: https://cvmrepo.web.cern.ch/cvmrepo/preload/cvmfs_preload
    dest: "{{ cvmfs_preload_path }}/cvmfs_preload"
    owner: root
    group: root
    mode: 755
  when: cvmfs_preload_install | bool
```

from https://github.com/galaxyproject/ansible-cvmfs/blob/master/tasks/client.yml , which still use a standard module ([get_url](https://docs.ansible.com/ansible/2.9/modules/get_url_module.html)), Jinja2 with a greppable variable, and a `when` condition.

Reply:
I'll make the change

Reply:
@nsoranzo I've updated the example; could you check the wording, please?